packages/connector/package.json

```diff
@@ -43,8 +43,10 @@
   "dependencies": {
     "@blockchain-lab-um/masca-types": "*",
     "@blockchain-lab-um/utils": "*",
+    "@didtools/pkh-ethereum": "0.0.2",
```

Comment:
Do we need this package ?

Reply:
Yes. Its used to for authorizing did session